contexts/ConfigContext.tsx

```diff
@@ -21,12 +38,22 @@ const defaultConfig = {
 
 export const ConfigContext = createContext<RuntimeConfig>(defaultConfig)
 
-export async function fetchConfigFromAPI(): Promise<RuntimeConfig> {
+export function getStaticConfig(): StaticConfig {
+  // Should we set sensible defaults here?
+  return {
+    avatarUrl: publicRuntimeConfig.avatarUrl,
+    flowInitAccountsNo: publicRuntimeConfig.flowInitAccountsNo,
+    tokenAmountFLOW: publicRuntimeConfig.tokenAmountFLOW,
+    tokenAmountFUSD: publicRuntimeConfig.tokenAmountFUSD,
```

Comment:
As per the Next.js docs, we should use plain environment variables (i.e. `process.env`) for build-time static config.

https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables